monai/networks/nets/resnet.py

```diff
@@ -14,14 +14,23 @@
 from collections.abc import Callable
 from functools import partial
 from typing import Any
+import logging
+
 
 import torch
 import torch.nn as nn
 
 from monai.networks.layers.factories import Conv, Norm, Pool
 from monai.networks.layers.utils import get_pool_layer
-from monai.utils import ensure_tuple_rep
+from monai.utils import ensure_tuple_rep, optional_import
 from monai.utils.module import look_up_option
+from monai.networks.utils import get_pretrained_resnet_medicalnet
+
+Path, _ = optional_import("pathlib", name = "Path")
+re, _ = optional_import("re")
```

Comment:
pathlib and re are both Python built-in libraries and do not require an optional import.